### PR TITLE
FBP-56 disable registration to prevent someone to register through Postman etc

### PR DESCRIPTION
**Pull request by UvgenGen**
[FBP-56](https://youtrack.raccoongang.com/issue/FBP-56) Remove register button from https://aicourses.polytechnic.bh/
- disable registration to prevent someone to register through Postman etc

### CODE REVIEW

File: common/djangoapps/student/views.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 from django.core.context_processors import csrf
 from django.core import mail
 from django.core.urlresolvers import reverse, NoReverseMatch, reverse_lazy
+from django.core.exceptions import PermissionDenied
```
Comment (sendr) [inline common/djangoapps/student/views.py:27]: import order

File: common/djangoapps/student/views.py [hunk 2]
```diff
@@ -1545,6 +1546,10 @@ def _do_create_account(form, custom_form=None):
 
     Note: this function is also used for creating test users.
     """
+    # feature implemented in edx
```
Comment (sendr) [inline common/djangoapps/student/views.py:1549]: can you please describe why you raise this error?

Comment (UvgenGen) [reply common/djangoapps/student/views.py:1549]: because this is how edx does it. might be better to replace it with 404?

Comment (sendr) [reply common/djangoapps/student/views.py:1549]: I mean describe it in the comment to make it more clear, for example
`To disable a user creation, this part is taken from the future releases of the platform`, something like that))

**Review by sendr — APPROVED** [review]

**Pull request merged**